modules/bookDesplay.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
+export default class Display {
+    books = [];
+
+    booksContainer = document.getElementById('containerBooks');
+
+    constructor() {
+      if (localStorage.getItem('books')) {
+        // eslint-disable-next-line no-undef
+        this.books = JSON.parse(localStorage.getItem('books')).map((book) => new Book(book.title, book.author, book.id));
+      }
+    }
+
+    addBook(newBook) {
+      this.books.push(newBook);
+      this.render();
+      this.saveBooks();
+    }
```

Comment:
- [ ] Kindly use `ES6` **_arrow syntax_** for all functions in this class. Remember that the requirements state that all functions should be written in `ES6`. Kindly implement this change. 👍